src/ragger/firmware/touch/layouts.py

```diff
@@ -100,15 +78,27 @@ class FullKeyboardSpecialCharacters2(_FullKeyboardSpecialCharacters):
 
 
 # Center Info
-class TappableCenter(_Layout):
+class TappableCenter(Element):
 
     def tap(self):
         self.client.finger_touch(*self.positions)
 
 
+class KeyboardConfirmationButton(Element):
+    """
+    This layout is to be used as the confirmation button when coupled with a keyboard.
+
+    On Stax devices, the screen is high enoug that this is equivalent to TappableCenter, however
```

Comment:
Minor, typo: `enough`

Reply:
Good catch. Weird codespell didn't flagged it :thinking: